### PR TITLE
[WIP] updated changelog

### PR DESCRIPTION
**Pull request by mkurek**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,24 +1,61 @@
...
 
-* New look for menu.
-* CORE: Added new plugin - proxmox.
-* CORE: Added new resource to API - PowerDNSRecord.
-* CORE: Added support for regions (used in Assets Module).
-* Bugfixes.
+New features
+************
+* Added data center and racks visualization.
+* Ralph required ralph_assets package.
```
Comment (vi4m) [inline CHANGES.rst]: requires

File: CHANGES.rst [hunk 2]
```diff
@@ -1,24 +1,61 @@
...
+* Improved Xen detection.
+* New CiscoSSHHandler added to provide better support for Cisco switches.
```
Comment (vi4m) [inline CHANGES.rst]: better ssh communication with cisco switches

File: CHANGES.rst [hunk 3]
```diff
@@ -1,24 +1,61 @@
...
+* Added possibility to store databases in Ralph and better support for storing load balancers virtual servers.
+* Updated docs for Don Pedro agent and Ralph's installation process
```
Comment (vi4m) [inline CHANGES.rst]: . at the end

File: CHANGES.rst [hunk 4]
```diff
@@ -1,24 +1,61 @@
...
+
```
Comment (vi4m) [inline CHANGES.rst]: We've moved to the rtd. Docker is the official way to easy install Ralph

File: CHANGES.rst [hunk 5]
```diff
@@ -1,24 +1,61 @@
...
+    - Puppet plugin can get data from puppetdb-api.
+    - 'Force autoscan' is now triggered only when SNMP name is missing.
+    - Got rid of proxmoxer, which gives different results for proxmox 2 vs 3.
+* DEPLOY:
+    - Added two missing features to deploy/clean plugin: The hostname is now copied from deployment and copying venture/role from deployment
+    - Changed the required permission for VM creation - You need core access to create a VM via API.
+    - Added new entry point to API to provide possibility to change IP address after deployment.
+* GUI:
+    - Added management IP in 'Addresses' tab (device view)
+    - Scan form - now it's possible to add some new components to existing device.
```
Comment (vi4m) [inline CHANGES.rst]: to the ...

File: CHANGES.rst [hunk 6]
```diff
@@ -1,24 +1,61 @@
...
+    - Blocked ability to add management IP via addresses form.
```
Comment (vi4m) [inline CHANGES.rst]: remove it, obsolete

File: CHANGES.rst [hunk 7]
```diff
@@ -1,24 +1,61 @@
...
+* CMDB:
+    - Set CI.state default value to ACTIVE.
+* OTHER:
+    - Unified top bar with scrooge.
```
Comment (vi4m) [inline CHANGES.rst]: The design of top bar is now in line with the Scrooge UI

File: CHANGES.rst [hunk 8]
```diff
@@ -1,24 +1,61 @@
...
+    - Scrooge API: Databases and Load balancer virtual servers added
+    - Added department to search form.
+    - Functionality to inject custom tracking code.
+    - Read-only management ip shown in addresses tab.
```
Comment (vi4m) [inline CHANGES.rst]: remove it, now!

File: CHANGES.rst [hunk 9]
```diff
@@ -1,24 +1,61 @@
...
+    - Admin panel: changing management and management ip addresses blocked.
+    - Makeconf add all pluggable apps now.
```
Comment (vi4m) [inline CHANGES.rst]: ralph makeconf set pluggable apps to the reasonable defaults

File: CHANGES.rst [hunk 10]
```diff
@@ -1,24 +1,61 @@
...
+
+Minor improvements
+******************
+* SCAN:
+    - Detaching logical subdevices not found by scan: Previously only physical subdevices not found by scan were detached. Extended this feature on logical children
+    - Detect Dell family in scan, when headers not contains `Server`
+    - Puppet-plugin: added expanduser call on cert paths.
+    - Puppet plugin can get data from puppetdb-api.
+    - 'Force autoscan' is now triggered only when SNMP name is missing.
+    - Got rid of proxmoxer, which gives different results for proxmox 2 vs 3.
```
Comment (vi4m) [inline CHANGES.rst]: remove it

File: CHANGES.rst [hunk 11]
```diff
@@ -1,24 +1,61 @@
...
+    - Set CI.state default value to ACTIVE.
+* OTHER:
+    - Unified top bar with scrooge.
+    - Scrooge API: Databases and Load balancer virtual servers added
+    - Added department to search form.
+    - Functionality to inject custom tracking code.
+    - Read-only management ip shown in addresses tab.
+    - Admin panel: changing management and management ip addresses blocked.
+    - Makeconf add all pluggable apps now.
+    - Added the management_ip property to device.
```
Comment (vi4m) [inline CHANGES.rst]: remove it

File: CHANGES.rst [hunk 12]
```diff
@@ -1,24 +1,61 @@
...
+    - For devices with assigned Asset (only without is_blade flag) there are no more possibility to change position.
```
Comment (vi4m) [inline CHANGES.rst]: there is no longer

File: CHANGES.rst [hunk 13]
```diff
@@ -1,24 +1,61 @@
...
+    - Adding the possibility of mapping the country to user profile through the attribute in the LDAP.
```
Comment (vi4m) [inline CHANGES.rst]: You can now set user's country using LDAP attribute

File: CHANGES.rst [hunk 14]
```diff
@@ -1,24 +1,61 @@
...
+    - Add Powerdns Record resource to REST API.
```
Comment (vi4m) [inline CHANGES.rst]: Added

File: CHANGES.rst [hunk 15]
```diff
@@ -1,24 +1,61 @@
...
+
+Fixes
+*****
+* Fix for MAC detecting in IDRAC plugin.
+* Parent and management fields are now readonly when an asset is assigned. Before this change we could't save form
+* Updated parse_lshw(..) to work with puppetdb_api. Used better msg when no facts found.
+* Fixed docker installation.
+* Changing management IP via Scan form when asset is assigned - blocked. Fix for assets saving on Scan form.
```
Comment (vi4m) [inline CHANGES.rst]: remove it

File: CHANGES.rst [hunk 16]
```diff
@@ -1,24 +1,61 @@
...
+* Scan form create IPAddress object when does not exist.
```
Comment (vi4m) [inline CHANGES.rst]: scan form now creates

**Comment by vi4m** [comment]
Squash yourself please.

**Comment by vi4m** [comment]
And go for production.

**Comment by mkurek** [comment]
@vi4m: squashed

**Comment by vi4m** [comment]
wykopy lecimy na główną.

**Pull request merged**